docs/docs-source/docs/modules/get-started/pages/setup-k8s-cluster.adoc

```diff
@@ -0,0 +1,77 @@
+= Setting up a GKE cluster
+:toc:
+:toc-title: ON THIS PAGE
+:toclevels: 2
+
+include::ROOT:partial$include.adoc[]
+
+This page walks you through the important steps to get Cloudflow up and running. For more detailed instructions on how to use or build your own Cloudflow installer, refer to the https://github.com/lightbend/cloudflow/tree/master/installer[installer section] of the GitHub repo.
+
+To create a new cluster, clone the https://github.com/lightbend/cloudflow/[repo] in your local system by selecting the latest release branch or download the latest release file from https://github.com/lightbend/cloudflow/releases[here], go to the `installer/` folder and follow the instructions below.
+
+* Run `gcloud init` if you are creating a GKE cluster and this is your first time doing so. This command initializes the gcloud system on your local and performs necessary settings of gcloud credentials. You can have a look at the details in the https://cloud.google.com/sdk/gcloud/reference/init[GKE documentation].
```

Comment:
here, we give instructions for `gcloud` but a bit later, we offer the option to create a GKE, AKS, or, EKS cluster.
To be consistent, we either add login sections for each of them or we just refer to 'login to your cluster' and maybe add links to the (external) gke, ask, and, eks login instruction pages.